EngineeringModel/ViewModels/ElementDefinitionBrowser/ElementDefinitionsBrowserViewModel.cs

```diff
@@ -367,31 +391,39 @@ public override void ComputePermission()
                 return;
             }
 
+            this.CanCopyToClipboard = true;
```

Comment:
this will not be needed I think.

Reply:
Since Copy To clipboard will be a common always present menu the cancopytoclibpbaord has been removed. Please see the changes